subsys/net/l2/can/can_l2.c

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Actually, I don't see the need for a CAN L2. Current can device could just use the dummy l2.